src/pages/FlowRun/Details-Tile.vue

```diff
@@ -218,6 +218,20 @@ export default {
           <v-list-item dense>
             <v-list-item-content>
               <v-list-item-subtitle class="text-caption">
+                <v-row no-gutters>
+                  <v-col cols="6">
+                    Version
```

Comment:
Since runs have a version field as well, I think it'd be clearer to say something like this instead:
```suggestion
                    {{ flowRun.flow.name }} version
```